src/packages/switch/doc.zh-TW.md

```
@@ -34,61 +34,71 @@ import { Switch } from '@nutui/nutui-react'
-| activeText | 打開時文字描述 | `string` | `-` |
-| inactiveText | 關閉時文字描述 | `string` | `-` |
-| onChange | 切換開關時觸發 | `onChange:(value: boolean, event: Event)` | `-` |
+| activeText | 打開時文字描述 | `ReactNode` | `-` |
+| inactiveText | 關閉時文字描述 | `ReactNode` | `-` |
+| onChange | 切換開關時觸髮 | `onChange:(value: boolean, event: Event)` | `-` |
 
 ## 主題定製
 
 ### 樣式變量
 
 組件提供了下列 CSS 變量，可用於自定義樣式，使用方法請參考 [ConfigProvider 組件](#/zh-CN/component/configprovider)。
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)